api/envoy/extensions/access_loggers/wasm/v3/wasm.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+package envoy.extensions.access_loggers.wasm.v3;
+
+import "envoy/extensions/wasm/v3/wasm.proto";
+
+import "udpa/annotations/status.proto";
+import "udpa/annotations/versioning.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.access_loggers.wasm.v3";
+option java_outer_classname = "WasmProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: Wasm access log]
+
+// Custom configuration for an :ref:`AccessLog <envoy_api_msg_config.accesslog.v3.AccessLog>`
+// that calls into a WASM VM. Configures the built-in *envoy.access_loggers.wasm*
```

Comment:
Do we still care about the old style names in v3? I.e. isn't type URL sufficient?

Reply:
What are the old style names ?  I copied this comment from api/envoy/extensions/access_loggers/file/v3/file.proto.

If I mark them not-implemented-yet then the internal links and documentation links fail because it doesn't generate the docs.  Do you want me to remove the docs?

Reply:
I'm suggesting that you adopt the same pattern we see in places such as https://github.com/envoyproxy/envoy/blob/master/api/envoy/extensions/compression/gzip/compressor/v3/gzip.proto#L17. We actually still have the old name there.. but the key difference is that nobody needs to use that to configure the gzip filter, they can put arbitrary content in the extension name. This is actually true for all filters in v3, the docs are trailing.

Reply:
Done. PTAL